src/algebra/hom/group.lean

```diff
@@ -1236,3 +1236,140 @@ protected lemma commute.map [mul_hom_class F M N] (h : commute x y) (f : F) :
 h.map f
 
 end commute
+
+section coe
+
+/-! ### Coercions as bundled morphisms
+
+The classes `coe_mul_hom`, `coe_monoid_hom`, etc. state that the coercion map `↑` (a.k.a. `coe`)
+is a homomorphism.
```

Comment:
```suggestion
The classes `coe_mul_hom`, `coe_monoid_hom`, etc. state that the coercion map `↑` (a.k.a. `coe`)
is a homomorphism.
These classes are unbundled (they take an instance of `has_lift_t R S` as a parameter, rather than
extending `has_lift_t` or one of its subclasses) for two reasons:

 * We wouldn't have to introduce new classes that handle transitivity (and probably cause diamonds)
 * It doesn't matter whether a coercion is written with `has_coe` or `has_lift`, you can give it
   a homomorphism structure in exactly the same way.
```